source/features/remove-projects-tab.tsx

```diff
@@ -77,5 +60,15 @@ features.add({
 		pageDetect.isUserProfile,
 		pageDetect.isOrganizationProfile
 	],
-	init
+	exclude: [
+		canUserEditRepo,
+		canUserEditOrganization
+	],
+	init: removeProjectsTab
+}, {
+	include: [
+		pageDetect.isRepo,
+		pageDetect.isOrganizationProfile
+	],
+	init: onetime(addNewProjectLink)
```

Comment:
I think, but not totally sure, that you can just use this since it's outside the ajaxed area
```suggestion
	repeatOnAjax: false
	init: addNewProjectLink
```